packages/server/package.json

```diff
@@ -12,7 +12,7 @@
     "db:seed": "knex seed:run",
     "test": "mocha __tests__/*.test.js",
     "test:db": "__tests__/db/run-tests.sh",
-    "test:api": "NODE_ENV=test SUPPRESS_EMAIL=TRUE node src > test.log & NODE_ENV=test mocha --require __tests__/api/fixtures.js __tests__/api/*.test.js && pkill -fn 'node src'",
+    "test:api": "NODE_ENV=test SUPPRESS_EMAIL=TRUE node src > test.log & NODE_ENV=test mocha --require __tests__/api/fixtures.js __tests__/api/*.test.js; exit_code=$?; pkill -fn 'node src'; exit $exit_code",
```

Comment:
This is pretty messy! A better pattern might be to use before/after each/all hooks in the tests to start and stop the dev server on a random port.

We have some nice tooling in Univaf for this (although ours is designed for Jest instead of Mocha, so it might require some slight tweaks to work here):

https://github.com/usdigitalresponse/univaf/blob/2fca59d558de365d84057bf09892d5e73e891e71/server/test/lib.ts#L18-L47

Use it like so:

https://github.com/usdigitalresponse/univaf/blob/2fca59d558de365d84057bf09892d5e73e891e71/server/test/api.edge.test.ts#L20-L36